### PR TITLE
create conmon sockets when getting their paths

### PR DESCRIPTION
**Pull request by baude**
when using the getattachsockets endpoint, which returns the sockets needed
to create and use a terminal, we should check if the container is just in the
configured state.  if so, we need to perform a container init to have conmon
create the required sockets so we can attach to them prior to starting the container.

Signed-off-by: baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/varlinkapi/containers.go [hunk 1]
```diff
@@ -462,6 +462,20 @@ func (i *LibpodAPI) GetAttachSockets(call ioprojectatomicpodman.VarlinkCall, nam
...
 	}
+
+	status, err := ctr.State()
+	if err != nil {
+		return call.ReplyErrorOccurred(err.Error())
+	}
+
+	// If the container hasn't been run, we need to run init
+	// so the conmon sockets get created.
+	if status == libpod.ContainerStateConfigured {
```
[the remark this answers is not in the document]
Comment (baude) [reply pkg/varlinkapi/containers.go]: fixed

**Comment by jwhonce** [comment]
@baude Unit Tests restored.  +1  Doing more testing now.

**Comment by baude** [comment]
bot, retest this please

**Comment by jwhonce** [comment]
@rh-atomic-bot r+

**Comment by mheon** [comment]
LGTM

**Pull request closed without merging**